packages/server/modules/pwdreset/rest/index.ts

```diff
@@ -6,24 +7,28 @@ import { Express } from 'express'
 export default function (app: Express) {
   // sends a password recovery email.
   app.post('/auth/pwdreset/request', async (req, res) => {
+    const boundLogger = logger.child({ endpoint: 'auth/pwdreset/request' })
```

Comment:
@iainsproat Do we really want all of these to be logged as errors? The errors thrown can easily be validation errors (e.g. invalid/expired tokens)